pkgs/applications/networking/browsers/lynx/default.nix

```diff
@@ -2,6 +2,7 @@
 , fetchurl, pkg-config, ncurses, gzip
 , sslSupport ? true, openssl ? null
 , nukeReferences
+, fetchpatch
```

Comment:
Either we fetch the patch or we vendor it. Fetching is preferred.

Reply:
removing.

Reply:
Removed.